NewForce-Cohort-9/cider-falls-park-sombrero-squad
# Il squad - destination stylings, click event, and other formatting

Changes

- added click event in destination section 
- swapped positions of 1st and 3rd items in database for destinations
- swapped logo
- added footer
- formatting for destinations section
- updated header to Cider Falls Park 


To test: 
- switch to branch il-squad
- see if everything loads & click event clicks
